cli/azd/pkg/sqldb/sqldb.go

```diff
@@ -43,9 +43,11 @@ func (c *sqlDbClient) ConnectionString(ctx context.Context, subId, rgName, serve
 		return "", fmt.Errorf("failed getting server '%s' for resource group '%s'", serverName, rgName)
 	}
 
-	serverDomain := *res.Server.Properties.FullyQualifiedDomainName
+	if res.Server.Properties == nil {
+		return "", fmt.Errorf("failed getting server properties from server '%s'", serverName)
+	}
 
-	if serverDomain == "" {
+	if res.Server.Properties.FullyQualifiedDomainName == nil {
```

Comment:
It's still worth checking for empty and failing, since that isn't a valid endpoint either.

```suggestion
	if res.Server.Properties.FullyQualifiedDomainName == nil || len(*res.Server.Properties.FullyQualifiedDomainName) == 0 {
```